cmd/kubeadm/app/util/pubkeypin/pubkeypin_test.go

```diff
@@ -48,7 +48,6 @@ aH+sfuxT8xeTPb3kjzF9eJTlnEquUDLM
 -----END CERTIFICATE-----`
 
 // expectedHash can be verified using the openssl CLI:
-// openssl x509 -pubkey -in test.crt openssl rsa -pubin -outform der 2>&/dev/null | openssl dgst -sha256 -hex
```

Comment:
the line above should now end up with `.` instead of `:`
applies to the other changes too.

Reply:
Thanks for the review. i will updated in a while.